it/conduct/index.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: page
+title: "Il codice di condotta della comunità di Ruby"
+lang: it
+---
+Il presente codice di condotta è stato scelto basandosi su una proposta di bozza per il codice di condotta di PostgreSQL, per mantenere la collaborazione nella comunità degli sviluppatori Ruby sicura e produttiva.
+Ogni cosa collegata alla comunità di Ruby (conferenze ecc.) dovrebbe avere il suo codice di condotta.
+{: .summary}
+
+Questo documento fornisce delle linee guida di comunità per ottenere un luogo sicuro, dove c'è rispetto, produttivo e collaborativo per ogni persona che voglia contribuire alla comunità di Ruby. Si applica a tutti gli spazi collaborativi, come i canali di comunicazione comunitaria (mailing lists, patch inviate, commenti ai commit ecc.).
```

Comment:
> dove c'è rispetto 

I think `rispettoso` is better since it does not break less the sentence flow

Reply:
> come i canali di comunicazione comunitaria

`comunitaria` sounds a bit weird in this context. Maybe we can just remove it or use `della community`, what do you think?

Reply:
@kennyadsl Solo che per l'ambiente non può essere rispettoso perché l'ambiente effettivamente non può rispettare, sono d'accordo con te sul fatto che suoni meglio.
Per quanto riguarda della community non credo ci siano problemi a cambiare così, quindi se ti sembra suonare meglio, procedo.